### PR TITLE
feat(Mappers): add booleanMapper into mappers

### PR DESCRIPTION
**Pull request by machadolucasvp**
#53 
When accepted, if so, can please label this PR with _hacktoberfest-accepted_ :smile: ?

### CODE REVIEW

**Comment by machadolucasvp** [comment]
@Siemienik what you think about create a `utils` folder ? problably `removeTextAfterValidNumber` will be used in #46 and #48

File: src/mappers/booleanMapper.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { ValueMapper } from '../abstracts/ValueMapper';
+
+const removeTextAfterValidNumber = (value: string): string | undefined =>
```
Comment (Siemienik) [inline src/mappers/booleanMapper.ts:3]: > @Siemienik what you think about create a `utils`/`helpers` folder ? problably `removeTextAfterValidNumber` will be used in #46 and #48

This util is redundant if you use `Number.parseFloat`

image

**Review by Siemienik — CHANGES_REQUESTED** [review]
Please to use `.parseFloat` instead of `removeTextAfterValidNumber`

**Comment by Siemienik** [comment]
> When accepted, if so, can please label this PR with _hacktoberfest-accepted_ 😄 ?

It isn't required if I merge PR into master and I will do it soon after resolving: https://github.com/Siemienik/xlsx-import/pull/54#discussion_r507620400

File: src/mappers/booleanMapper.ts (hunk 1 above)
Comment (machadolucasvp) [reply src/mappers/booleanMapper.ts:3]: Aah okay :smiley:, i had no idea that parseFloat do that under the hood

**Review by Siemienik — APPROVED** [review]
LGTM

**Pull request merged**